src/components/Signup.tsx

```diff
@@ -0,0 +1,161 @@
+import { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+
+import Button from '@mui/material/Button';
+import TextField from '@mui/material/TextField';
+import Typography from '@mui/material/Typography';
+import Container from '@mui/material/Container';
+import Link from '@mui/material/Link';
+import Grid from '@mui/material/Grid';
+import Box from '@mui/material/Box';
+import ToggleButton from '@mui/material/ToggleButton';
+import ToggleButtonGroup from '@mui/material/ToggleButtonGroup';
+import { createUserWithEmailAndPassword } from '../api/auth';
+
+import { createFirebaseDao } from '../api/dao';
+
+function Signup() {
+    const navigate = useNavigate();
+    const [firstName, setFirstName] = useState('');
+    const [lastName, setLastName] = useState('');
+    const [password, setPassword] = useState('');
+    const [email, setEmail] = useState('');
+    const [address, setAddress] = useState('');
+    const [userType, setUserType] = useState('senior');
```

Comment:
Not declaring a type here means that `userType` will be inferred as a string.  (see screenshot below)

<img width="822" alt="Screen Shot 2022-10-03 at 8 32 28 PM" src="https://user-images.githubusercontent.com/15661253/193728552-a5aba147-0e98-4c60-a173-0247d6f4240e.png">

Making it string means 'foo', 'bar' and some other strings are valid values. We don't want that though, because `userType` should only be a `nurse` or a `senior`. 

Might be worth to declare an enum:

```
export enum UserType {
  Nurse = 'nurse',
  Senior = 'senior
}
```
or just a simple `type` declaration.

```
export type UserType = 'nurse' | 'senior';
```

Reply:
why does it show that I'm not using `userType` if I am using it?

Reply:
Oh that screenshot was from my editor